### ISSUE TITLE
Can not access the chat page....

### ISSUE THREAD

**Issue by saracura** [justdjango/justchat#1]
Can not access the chat page....whenever I get git clone I can not access the chat only the Django Rest Framework page and not the chat itself.

**Comment by saracura** [comment]
I can see I have no static files to all the js files...

**Comment by TeplrGuy** [comment]
try run `python manage.py collectstatic`

**Comment by justdjango** [comment]
@saracura is your issue resolved?

**Comment by saracura** [comment]
Nota yet. It  seems the problem has something to do with the parcel server.
It installed but the server does not run. My server is hosted at Digital
Ocean. The command line appears to run but whenever I list the processes I
do not see It and if I scan the IP the port also does not show. If you have
any clue.  Thanks for the concern.  Kind regards.

Em Dom, 25 de nov de 2018 03:17, JustDjango <notifications@github.com
escreveu:

**Comment by mattfreire** [comment]
You shouldn't be trying to run a parcel server on digital ocean. You must build the frontend and use those as the static files for your django project. Then you can serve your django project with the build react frontend.

**Comment by saracura** [comment]
Thanks for the help so far.
Found out my digitalocean nodejs version was to old(so I updated to latest), then for some reason it keeps trying to call parcel as below:

 cd frontend/
(chatCompleto) root@django-digio:/home/django/python3/projetos/chatCompleto/justchat/frontend# ls
index.html  node_modules  package.json  package-lock.json  src
(chatCompleto) root@django-digio:/home/django/python3/projetos/chatCompleto/justchat/frontend# npm start

> frontend@1.0.0 start /home/django/python3/projetos/chatCompleto/justchat/frontend
> parcel index.html

Server running at http://localhost:1234
⚠️  Could not load existing sourcemap of "src/node_modules/mutationobserver-shim/dist/mutationobserver.min.js".
⠹ Building hmr-runtime.js...Killed
npm ERR! code ELIFECYCLE
npm ERR! errno 137
npm ERR! frontend@1.0.0 start: `parcel index.html`
npm ERR! Exit status 137
npm ERR!
npm ERR! Failed at the frontend@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-11-30T15_39_47_475Z-debug.log
